dbms/src/Flash/Mpp/MPPTunnel.cpp

```diff
@@ -87,34 +86,13 @@ MPPTunnel::~MPPTunnel()
     SCOPE_EXIT({
         GET_METRIC(tiflash_object_count, type_count_of_mpptunnel).Decrement();
     });
-    try
-    {
-        close("");
-    }
-    catch (...)
-    {
-        tryLogCurrentException(log, "Error in destructor function of MPPTunnel");
-    }
+    close("", true);
```

Comment:
Suggest to keep the try/catch protection, or should introduce mechanisms like "no except" to guarantee that close won't throw any exceptions.

Reply:
OK, I will add no except for `MPPTunnel::close()`